packages/core/src/components/time-picker-rework/time-picker-rework.tsx

```diff
@@ -0,0 +1,328 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Siemens AG
+ *
+ * SPDX-License-Identifier: MIT
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {
+  Component,
+  Event,
+  EventEmitter,
+  h,
+  Host,
+  Method,
+  Prop,
+  State,
+  Watch,
+} from '@stencil/core';
+import dayjs, { UnitType } from 'dayjs';
+import customParseFormat from 'dayjs/plugin/customParseFormat';
+import { DateTimeCardCorners } from '../date-time-card/date-time-card';
+
+export type TimePickerCorners = DateTimeCardCorners;
+
+dayjs.extend(customParseFormat);
+
+interface TimePickerDescriptor {
+  unit: 'hours' | 'minutes' | 'seconds';
+  placeholder: string;
+  hidden: boolean;
+}
+
+interface TimeOutputFormat {
+  hours: string;
+  minutes: string;
+  seconds: string;
+}
+
+/**
+ * @internal
+ */
+@Component({
+  tag: 'ix-time-picker-rework',
+  styleUrl: 'time-picker-rework.scss',
+  shadow: true,
+})
+export class TimePickerRework {
+  /**
+   * Format of time string
+   *
+   * @since 1.1.0
+   */
+  @Prop() format: string = 'HH:mm:ss';
+
+  /**
+   * Corner style
+   */
+  @Prop() corners: TimePickerCorners = 'rounded';
+
+  /**
+   * Controls the visual presentation and styling of the component when it is displayed as a standalone element
+   */
+  @Prop() standaloneAppearance: boolean = true;
+
+  /**
+   * Show hour input
+   */
+  @Prop() showHour = false;
+
+  /**
+   * Show minutes input
+   */
+  @Prop() showMinutes = false;
+
+  /**
+   * Show seconds input
+   */
+  @Prop() showSeconds = false;
```

Comment:
As from developer perspective if i use the time picker with the following code:

```
        <ix-time-picker-rework
          showHour={true}
          showMinutes={false}
        ></ix-time-picker-rework>
```

i will get this as an result:
![image](https://github.com/siemens/ix/assets/2144313/9d0dfafa-8eb0-4a00-aa1d-611e1da1b064)

If you then try this code:
```
        <ix-time-picker-rework showMinutes={false}></ix-time-picker-rework>
```

you would expect to see nothing right? But this is not the case know you see all hours, minutes and second files.

I think the default `false` makes the issue here if you replace it with `true` it should solve the problem. Right?